m4_sessions/1808-inning/mental_health_in_tech.md

```diff
@@ -1,30 +1,45 @@
-Austin Wiedenman
-Gavin Love
-Jamison Ordway
-Steve Schwedt
+Anten Skrabec
+Eric Weissman
+Tiffany Bachmann
+Whitney Burton
 
 ## Name of Session: 
 Mental Health in Tech Industry
 
 ## Session Description 
-
-1. Effects of Tech Industry culture on mental health conditions, ways to prevent burnout, and how we can support each other.
+Effects of Tech Industry culture on mental health conditions, ways to prevent burnout, and how we can support each other.
 
 ## Session Outcomes 
-
 1. An understanding of how issues of mental health are addressed, or not addressed, in the industy.
 2. Ways to find support and how they can support each other.
 
-## Session Outline 
+## Supplies
+- Poster Paper
+- Sticky Notes
+- Markers
 
-**(15 min)** Introduction(students)why they chose this session?/ how they implement self care?/ or other?, including presenters personal stories about their experiences here at Turing
-**(10 min)** video
-**(15 min)** small groups, reading differetn articles and discussions as groups.
-     *[Stress & Anxiety In The Tech Industry](https://risepsychology.com/blog/2017/10/30/stressintech)*
-     *[Devs and Depression](http://baugues.com/devs-and-depression)*
-     *[Are You More Than Okay: The State of Mental Health in Tech in 2016](https://modelviewculture.com/pieces/are-you-more-than-okay-the-state-of-mental-health-in-tech-in-2016)*
-     *[How should we talk about mental health?](https://ideas.ted.com/how-should-we-talk-about-mental-health/)*
-**(15 min)** large group discussion
-**(10 min)** looking forward, and strategies
-**(5 min)** students write in their journal about some of their own strategies and ways they can help others.
+## Session Outline – Total: 70 Minutes
+1. **(20 min)** Gear-up Introduction
+  - **(5 min)** Facilitator Introductions
+    - Personal Stories/Experiences; Why do we think it's important?
+  - **(5 min)** Group Introductions
+    - Name, pronouns, etc.
+    - Why they chose this session?
+  - **(5 min)** Group Norms
+    - Why is it important to talk about mental health?
+2. **(10 min)** Pomodoro Break & 3 min Headspace Meditation
+3. **(15 min)** small groups of 3, reading different articles and discussions as groups.
+  - **(8 min)** Read different articles of a group individually
+    1. *[Stress & Anxiety In The Tech Industry](https://risepsychology.com/blog/2017/10/30/stressintech)*
+    2. *[Are You More Than Okay: The State of Mental Health in Tech in 2016](https://modelviewculture.com/pieces/are-you-more-than-okay-the-state-of-mental-health-in-tech-in-2016)*
+    3. *[How should we talk about mental health?](https://ideas.ted.com/how-should-we-talk-about-mental-health/)*
+  - Questions to think about
+    - What is your biggest take away from the article?
+    - What surprised you the most in the article?
+    - What kind of changes could you make in your own life?
+    - What are some other ideas/strategies for mental health support?
+  - **(7 min)** Disccusions in small groups
```

Comment:
How do you want to select groups?  Mix them up?  Count off?

Reply:
Count off by 3s